lib/dangermattic/plugins/ios_release_check.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Danger
+  # Plugin for performing iOS / macOS release-related checks in a pull request.
+  #
+  # @example Checking for changes in Core Data models on a release branch:
+  #          ios_release_check.check_core_data_model_changed
+  #
+  # @example Checking for modified Localizable.strings on a regular branch:
+  #          ios_release_check.check_modified_localizable_strings
+  #
+  # @example Checking for synchronization between release notes and App Store strings:
+  #          ios_release_check.check_release_notes_and_app_store_strings
+  #
+  # @see Automattic/dangermattic
+  # @tags ios, macos, process, release
+  #
+  class IosReleaseCheck < Plugin
+    LOCALIZABLE_STRINGS_FILE = 'Localizable.strings'
+    BASE_STRINGS_FILE = "en.lproj/#{LOCALIZABLE_STRINGS_FILE}".freeze
+
+    # Checks if an existing Core Data model has been edited in a release branch.
+    #
+    # @return [void]
+    def check_core_data_model_changed
+      warning = 'Do not edit an existing Core Data model in a release branch unless it hasn\'t been released to testers yet. ' \
+                'Instead create a new model version and merge back to develop soon.'
+
+      common_release_checks.check_file_changed(
+        file_comparison: ->(path) { File.extname(path) == '.xcdatamodeld' },
+        message: warning,
+        on_release: true
+      )
+    end
+
+    # Checks if any Localizable.strings file has been modified on a release branch, otherwise reporting a warning.
+    #
+    # @return [void]
+    def check_modified_localizable_strings_on_release
```

Comment:
I think it would be valuable to implement the same rule for Android (on `res/values-*/strings.xml` files) for the same reason as iOS?

Reply:
Agreed -- this wasn't originally in Peril iinm, but I do remember you mentioning this check in another discussion and I even started implementing it, I might have it stashed. Do you think it is a requirement for this PR to land or I can add a task in the iteration project to add it? It's not a lot in any case.